files/image_config/caclmgrd/caclmgrd

```diff
@@ -182,6 +182,10 @@ class ControlPlaneAclManager(object):
                 acl_rules = {}
 
                 for ((rule_table_name, rule_id), rule_props) in self._rules_db_info.iteritems():
+                    if not rule_props:
```

Comment:
>if not rule_props [](start = 20, length = 17)

Embed this if-block under inside below if-block?

Reply:
Done: https://github.com/Azure/sonic-buildimage/pull/2475/commits/739939ad1caa733ce80c584a9f386d3cdc98674b